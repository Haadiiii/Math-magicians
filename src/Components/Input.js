import React from "react";

export default class Input extends React.Component {
  render() {
    return (
      <div className="buttons">
        <input name="integers" placeholder="0" className="input" />
      </div>
    );
  }
}
