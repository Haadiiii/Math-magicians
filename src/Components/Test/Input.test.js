import React from "react";
import Input from "../Input";
import { render, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("render Without crashing", () => {
  render(<Input />);
});

it("matches snapshot for Input", () => {
  const tree = renderer.create(<Input></Input>).toJSON();
  expect(tree).toMatchSnapshot();
});