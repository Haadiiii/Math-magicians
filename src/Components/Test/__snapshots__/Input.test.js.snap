// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for Input 1`] = `
<div
  className="buttons"
>
  <div
    className="input"
  />
</div>
`;
